stacks/st2testing.yaml

```diff
@@ -7,6 +7,8 @@ defaults: &defaults
   puppet:
     facts:
       role: norole
+  private_networks:
+    - 192.168.60.40
```

Comment:
should be fine, since it's test... you won't be running multiple boxes at a time. was my reasoning, at least.